### PR TITLE
Routesdillon

### PR DESCRIPTION
**Pull request by Dillon-Armstrong**
- Implemented route to GET product and Styles on server.
- created requests.js and set up initial GET on render with useEffect.
- some bugs to work out in component but the requests file works. please look at the file and how it's built and let me know if you have questions or issues with it.

### CODE REVIEW

File: client/src/components/requests.js
Comment (lundas) [inline client/src/components/requests.js:1]: kinda nitpicky, but is there a reason to name this function "info" after it's been assigned to the product property?

**Review by lundas — COMMENTED** [review]
Going to hold off on approving so that @TammamTautou and @Benjamin-Cardon can look over as well before merge

See my one–albeit nitpicky–comment. Otherwise all looks good. Well done, bud.

Can update my routes to mirror these once merge is complete

**Review by Benjamin-Cardon — COMMENTED** [review]
I've looked over this. I don't think that the styles are going to touch my module, but I'm glad to know what you're doing.

File: client/src/components/requests.js
Comment (Dillon-Armstrong) [reply client/src/components/requests.js:1]: Yeah because the linter was yelling at me about 'unnamed function'. I just took another look and i can get around that with arrow function syntax. I can add the to my next PR.

**Review by TammamTautou — APPROVED** [review]
looks good

**Pull request merged**
